src/fides/api/models/privacy_experience.py

```diff
@@ -219,260 +561,41 @@ class PrivacyExperience(Base):
-        )
-
     @staticmethod
-    def create_default_experience_for_region(
-        db: Session, region: PrivacyNoticeRegion, component: ComponentType
-    ) -> PrivacyExperience:
-        """Creates an Experience for a given Component Type and Region and links
-        to default copy (ExperienceConfig)"""
-        experience_data: Dict = {
-            "region": region,
-            "component": component,
-        }
-        default_config: Optional[
-            PrivacyExperienceConfig
-        ] = PrivacyExperienceConfig.get_default_config(db, component)
-
-        if default_config:
-            experience_data["experience_config_id"] = default_config.id
-
-        return PrivacyExperience.create(
-            db=db,
-            data=experience_data,
-        )
-
-    @staticmethod
-    def get_experience_by_region_and_component(
+    def get_experiences_by_region_and_component(
         db: Session, region: str, component: ComponentType
-    ) -> Optional[PrivacyExperience]:
-        """Load an experience for a given region and component type"""
+    ) -> Query:
+        """Utility method to load experiences for a given region and component type"""
         return (
             db.query(PrivacyExperience)
+            .join(
+                PrivacyExperienceConfig,
+                PrivacyExperienceConfig.id == PrivacyExperience.experience_config_id,
+            )
             .filter(
                 PrivacyExperience.region == region,
-                PrivacyExperience.component == component,
-            )
-            .first()
-        )
-
-    @staticmethod
-    def get_overlay_and_privacy_center_experience_by_region(
-        db: Session, region: str
-    ) -> Tuple[Optional[PrivacyExperience], Optional[PrivacyExperience]]:
-        """Load both the overlay and privacy center experience for a given region
-
-        TCF overlays are not returned here.  This method is used in building experiences when Notices
-        are created, which is not applicable for TCF.
-        """
-        overlay_experience: Optional[
-            PrivacyExperience
-        ] = PrivacyExperience.get_experience_by_region_and_component(
-            db=db, region=region, component=ComponentType.overlay
-        )
-
-        privacy_center_experience: Optional[
-            PrivacyExperience
-        ] = PrivacyExperience.get_experience_by_region_and_component(
-            db=db, region=region, component=ComponentType.privacy_center
-        )
-
-        return overlay_experience, privacy_center_experience
-
-    def unlink_experience_config(self, db: Session) -> PrivacyExperience:
-        """Removes the experience config
-
-        Note that neither the Experience Config or Experience are deleted; we're only removing
-        a FK if it exists.
-        """
-        return self.update(  # type: ignore[return-value]
-            db,
-            data={"experience_config_id": None, "experience_config_history_id": None},
-        )
```

Comment:
We previously created an Experience for a region or a component that could be linked or unlinked from various ExperienceConfigs.  Now we just delete the Experience if a region is removed from a PrivacyExperienceConfig.  As a result, PrivacyPreferences no longer have a FK to this object. By itself, it doesn't add information you don't have from the PrivacyExperienceConfig and user geography.